projects/experimental/components/avatar-stack/avatar-stack.module.ts

```diff
@@ -0,0 +1,14 @@
+import {NgModule} from '@angular/core';
+import {
+    TuiAvatarComponent,
+    TuiAvatarModule,
+} from '@taiga-ui/experimental/components/avatar';
+
+import {TuiAvatarStackComponent} from './avatar-stack.component';
+
+@NgModule({
+    imports: [TuiAvatarModule],
+    declarations: [TuiAvatarStackComponent],
+    exports: [TuiAvatarStackComponent, TuiAvatarComponent],
+})
+export class TuiAvatarStackModule {}
```

Comment:
It is not obvious from the documentation that we should import this new module.

Should we add description to [this example](https://taiga-ui--pr5090-experimental-32tiixoj.web.app/experimental/avatar#stacking)?

Why `TuiAvatarStackComponent` cannot be part of `TuiAvatarModule` ?